src/native/containers/dn-simdhash-specialization-declarations.h

```diff
@@ -58,9 +58,11 @@ DN_SIMDHASH_TRY_ADD (DN_SIMDHASH_T_PTR hash, DN_SIMDHASH_KEY_T key, DN_SIMDHASH_
 uint8_t
 DN_SIMDHASH_TRY_ADD_WITH_HASH (DN_SIMDHASH_T_PTR hash, DN_SIMDHASH_KEY_T key, uint32_t key_hash, DN_SIMDHASH_VALUE_T value);
 
+// result is an optional parameter that will be set to the value of the item if it was found.
```

Comment:
thanks! I'm thinking I should probably just write up an overall 'how to use simdhash' doc somewhere since the macros make the header not self-explanatory and there is some non obvious stuff